methods.py

```diff
@@ -904,9 +904,14 @@ def __getitem__(self, k: str):
                 defines=mono_defines,
             )
 
-        env["MSVSBUILDCOM"] = module_configs.build_commandline("scons")
-        env["MSVSREBUILDCOM"] = module_configs.build_commandline("scons vsproj=yes")
-        env["MSVSCLEANCOM"] = module_configs.build_commandline("scons --clean")
+        scons_cmd = "scons"
+        path_to_venv = os.getenv("VIRTUAL_ENV")
+        if path_to_venv:
+            scons_cmd = rf"{path_to_venv}\Scripts\scons.exe"
```

Comment:
it could be better to check if the file exists rather than assume it's there just because a virtualenv is active